sharedresource/v1alpha1/types_share.go

```diff
@@ -0,0 +1,33 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SharedResourceStatus contains the observed status of shared resource. Read-only.
+type SharedResourceStatus struct {
+	// conditions represents any observations made on this particular shared resource by the underlying CSI driver or Share controller.
+	// +patchMergeKey=type
+	// +patchStrategy=merge
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+
+// ResourceReferenceType represents a shared resource type
+type ResourceReferenceType string
```

Comment:
dead type?

Reply:
This is actually used in the openshift/csi-driver-shared-resource repository.

Reply:
> This is actually used in the openshift/csi-driver-shared-resource repository.

It seems more appropriate to move it into that repo.  It doesn't looking like an external API for clients

Reply:
removed in current push, and will be moved to other repo